sass/px-data-grid.scss

```diff
@@ -18,7 +22,6 @@ px-data-grid-cell-content-wrapper {
   min-height: 100%;
   display: flex;
   align-items: center;
-  background-color: var(--px-data-grid-cell-dragging-color, var(--px-data-grid-highlight-background-color, transparent));
```

Comment:
If you remove this you break D&D coloring.

Reply:
To see the issue, use highlight demo, allow reordering, and try to d&d column with highlights. With this change D&D color does not override highlight colors. Not sure if D&D works otherwise even.

Reply:
Oops, my bad. Reverted.

Reported yet another IE11 issue: https://github.com/vaadin/px-data-grid/issues/279